src/cli/README.md

```diff
@@ -0,0 +1,36 @@
+# Command Line Interface
+
+This is a simple command line interface for interacting with a Local Fulfillment app.
+
+# Usage
+
+## To start the command line interface:
+
+Run `npm run cli -- --app_path APP_PATH`, where `APP_PATH` is an absolute path to the entry point of your app (typically `bundle.js` if basing on the [Local Home SDK Sample](https://github.com/actions-on-google/smart-home-local))
+
+## To send an IDENTIFY intent
+
+Run `identify IDENTIFY --request_id REQUEST_ID --discovery_buffer DISCOVERY_BUFFER --device_id DEVICE_ID`
+
+- `REQUEST_ID` is the request id.
+- `DISCOVERY_BUFFER` is the discovery buffer, formatted as a hex string.
```

Comment:
How do we come up with this hex string?

Reply:
That would depend on the device being identified (it is sent directly to the `identifyHandler`).  For the sample, for example, it would be a `cbor` encoded value.